.github/workflows/rust.yml

```diff
@@ -70,6 +77,13 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v2
 
+      - name: Download Substrate
+        run: |
+            curl "https://releases.parity.io/substrate/x86_64-debian:stretch/latest/substrate/substrate" --output substrate --location
```

Comment:
I don't much like the repetition of this curl thing, but the alternate way of sharing artifacts between jobs works out to be about as verbose...